src/Appwrite/Migration/Migration.php

```diff
@@ -62,7 +62,8 @@ abstract class Migration
         '1.3.4' => 'V18',
         '1.3.5' => 'V18',
         '1.3.6' => 'V18',
-        '1.3.7' => 'V18',
+        //'1.3.7' => 'V18',
+        '1.3.7' => 'V19',
```

Comment:
Let's leave 1.3.7 as V18 and add 1.4.0 as V19, this will be for the next release

Reply:
Done!